### PR TITLE
adding file viewer 

### LINKED ISSUE #821: Add a context menu with options for "opening" an asset (or dandiset, folder?) on external services
original idea: https://github.com/dandi/dandiarchive/wiki/WiP:-dump-of-thoughts-on-%22services-registry%22

An [issue](https://github.com/MetaCell/nwb-explorer/issues/244) with metacell nwb viewer resolved , so we have no roadblock to start working on this feature to gain initial integrations. And bisweb might be just a CORR tune up away: https://github.com/dandi/dandi-infrastructure/pull/88 .

Similar implementation [is ongoing](https://github.com/datalad/datalad-deprecated/pull/27) for https://datasets.datalad.org and could be previewed in action for a sample dandiset at http://datasets-dev.datalad.org/?dir=/dandi/dandisets/000027/sub-RAT123 (hover over file to see ⠇)

Instead (or in addition) to a context menu it could be an action to take upon a click on an asset (https://github.com/dandi/dandiarchive/issues/780) but [I still think](https://github.com/dandi/dandiarchive/issues/780#issuecomment-885061015) we better reserve that for a "view" of some kind without resorting to external services.

also ref: https://github.com/dandi/dandi-infrastructure/issues/10 as a generic metacell deployment issue (not necessarily just linking out)

### PR DESCRIPTION
**Pull request by djarecka**
this is continuation of #849 

Adding file viewers (external services) to the files list as an option to see the file using external websites.

closes #821

### CODE REVIEW

**Comment by djarecka** [comment]
anyone knows which datasets have valid nwb files here: https://deploy-preview-854--gui-dandiarchive-org.netlify.app/#/ ?

**Comment by dchiquito** [comment]
I believe https://deploy-preview-854--gui-dandiarchive-org.netlify.app/#/dandiset/000018/draft/files?location=sub-anm372909%2F is valid, but I'm not sure.

**Comment by djarecka** [comment]
@dchiquito - thanks, this file will not show any service, since it's bigger than the maximal size I set, but will try to find a different one

File: src/views/FileBrowserView/PublishFileBrowser.vue [hunk 1]
```diff
@@ -173,6 +218,29 @@ const sortByName = (a, b) => {
...
 
+const EXTERNAL_SERVICES = [
+  {
+    name: 'Bioimagesuite/Viewer',
+    regex: '.nii(.gz)?$',
+    extensions: [
+      '.nii',
+      '.nii.gz',
+    ],
+    maxsize: 4e6,
```
Comment (mvandenburgh) [inline src/views/FileBrowserView/PublishFileBrowser.vue:229]: Should this be 4e9 (for 4 GB)? 4 MB seems pretty restrictive considering the typical file sizes on dandi

Comment (djarecka) [reply src/views/FileBrowserView/PublishFileBrowser.vue:229]: you're right it is too restrictive...

**Comment by djarecka** [comment]
the issue was with my `asset_id`, not that the files was wrong. 

However, even for 5MB it takes some time to load: 
https://deploy-preview-854--gui-dandiarchive-org.netlify.app/#/dandiset/000018/draft/files?location=sub-anm372909%2F

**Review by mvandenburgh — APPROVED** [review]
Looks good to me! Feel free to merge whenever you're ready @djarecka

**Pull request merged**